src/routes/namespaces/[namespace]/workflows/_workflows-summary-table.svelte

```diff
@@ -1,11 +1,46 @@
+<script lang="ts">
+  import {
+    workflowIdColumnWidth,
+    workflowTypeColumnWidth,
+  } from '$lib/stores/column-width-store';
+
+  // Need to measure column width since we can't use a tooltip and use overflow-hidden for text overflow
+  let workflowIdWidth: number = 0;
+  let workflowTypeWidth: number = 0;
+
+  $: {
+    workflowIdColumnWidth.set(workflowIdWidth);
```

Comment:
You don't need to do both the reactive statements and use the store. An `=` on a store prefixed with a `$` will do this automatically.

Reply:
👍